src/renderer/components/Slider.tsx

```diff
@@ -33,25 +35,40 @@ const Slider = ({
     setSliderValue(value);
   }, [value]);
 
-  const handleChangeValue = async (newValue: number) => {
-    await setValue(newValue);
-  };
+  const handleChangeValue = useCallback(
+    async (newValue: number) => {
+      await setValue(newValue);
+    },
+    [setValue]
+  );
 
   const handleInput = async (newValue: number) => {
     setSliderValue(newValue);
     handleChangeValue(newValue);
   };
 
+  const throttledSetValue = useThrottle(handleChangeValue, INTERVAL);
+
+  // Helpers for adjusting the preAmp gain value
+  const handleChangeValueWithThrottle = async (newValue: number) => {
```

Comment:
Why are we throttling this? I don't think UI changes should be throttled. As for the actual value change call (handleChangeValue), we already throttle this in FrequencyBand.tsx.

Reply:
OH mb, this was probably a change you undid in your PR relating to throttling. I got confused what changes I needed to keep when performing the rebase. Let me undo this.